modules/core/src/main/scala/org/scalasteward/core/data/Version.scala

```diff
@@ -124,6 +124,8 @@ final case class Version(value: String) {
 }
 
 object Version {
+  case class Update(currentVersion: Version, nextVersion: Version)
```

Comment:
**praise:** 👏🏼 👏🏼 

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org) standard</sub>